Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM node:10.11
+FROM node:10-alpine
```

Comment:
Do this to make sure the `apk add` command on line 4 works. (Needed for `dockerize`.)

Reply:
I'm muzzy on all this, but check out https://hub.docker.com/r/circleci/node  -- it has dockerize, and will work on circleci.   Maybe that's a worse solution than what you picked.  By my research here:  https://hub.docker.com/_/node/  we are already out of date on our node release.  :-P